src/components/app.jsx

```diff
@@ -65,8 +80,12 @@ const App = React.createClass({
                 </div>
                 <div className="col-md-9">
                   <p id="baseline">
-                    {Object.keys(variables).length} variables et {Object.keys(parameters).length} paramètres
-                    référencés pour modéliser le système socio-fiscal français.
+                    <FormattedMessage {...messages.header} />
+                    <br/>
+                    <br/>
+                    <small id="stats">
+                      <FormattedMessage {...messages.stats} values={{nbv: Object.keys(variables).length, nbp: Object.keys(parameters).length}}/>
```

Comment:
Why use cryptic 3-letters identifiers? `variablesCount` and `parametersCount` would be much more readable 😉